test/test_torch.py

```diff
@@ -4245,7 +4140,7 @@ def test_expand(self):
         self.assertRaises(RuntimeError, lambda: tensor2.expand(-1, -1))
 
         # test expanding empty to empty
-        self.assertEqual(torch.randn(()).expand(()), torch.randn(()))
+        self.assertEqual(torch.Tensor(0).expand((0,)), torch.Tensor(0))
```

Comment:
oh wow this is an ugly test.  Is this another incompatibility in that torch.randn(()) used to give an empty tensor but now gives a scalar?  I don't remember this on the incompatibility list.  As before I think we should avoid adding more torch.Tensor calls -- can we do torch.randn(0) here or something similar?

Reply:
I got rid of the torch.Tensor call. I'm not too worried about the change to randn. I don't think anyone is actually likely to use it in a way that will break.

Reply:
Ok, I added the randn thing to the list of incompatibilities under "minor"